music.js

```diff
@@ -176,6 +176,11 @@ document.getElementById("remote").onclick = () => {
 document.getElementById("upload").onclick = () => {
     document.getElementById("fileselector").click();
 }
+document.getElementById("toggletable").onclick = () => {
+    const table = document.getElementById("table");
+    const s = table.getAttribute('data-toggle-state');
```

Comment:
Heard of the data API?

Use table.dataset["toggleSet"] instead shall make things better

Reply:
Nice suggestion. Will be cleaner.